Source/DataSources/StaticGroundGeometryPerMaterialBatch.js

```diff
@@ -0,0 +1,398 @@
+define([
```

Comment:
This file is all @bagnell...

Reply:
@bagnell in case it makes things easier, this file was modified from `StaticGeometryPerMaterialBatch`